plugins/modules/system/xfconf.py

```diff
@@ -211,6 +211,11 @@ def _get(self):
     def state_absent(self):
         with self.runner('channel property reset', check_mode_skip=True) as ctx:
             ctx.run(reset=True)
+            self.vars.stdout = ctx.results_out
+            self.vars.stderr = ctx.results_err
+            self.vars.cmd = ctx.cmd
```

Comment:
`stdout` and `stderr` are [standard return values](https://docs.ansible.com/ansible/latest/reference_appendices/common_return_values.html#common-return-values), but `cmd` is not, so it should be documented in `RETURN`.